erpnext/projects/report/daily_time_log_summary/daily_time_log_summary.py

```diff
@@ -4,49 +4,92 @@
 from __future__ import unicode_literals
 import frappe
 from frappe import _
-from frappe.utils import flt
+from frappe.utils import flt,cint
 
 def execute(filters=None):
+	simplified_time_log = cint(frappe.db.get_value("Project Settings", None, "simplified_time_log"))
 	if not filters:
 		filters = {}
 	elif filters.get("from_date") or filters.get("to_date"):
 		filters["from_time"] = "00:00:00"
-		filters["to_time"] = "24:00:00"
```

Comment:
yes, necessary, this code is checking "Simplified Time Log" value and assigning different value to filters["to_time"]